kubernetes/cray-service/Chart.yaml

```diff
@@ -23,7 +23,7 @@
 #
 apiVersion: v2
 name: cray-service
-version: 8.2.2
+version: 8.2.3
```

Comment:
If we don't plan to deliver it to CSM 1.3, can we bump the minor or major version instead of the patch version? I'm currently working on a 1.3 issue, and plan to bump the patch version. The idea is to leave some space between release versions for patches.

Reply:
I can change it to whatever we want, it doesn't actually change anything though so the patch seemed fine as the api that was removed was just renamed basically.